grafana-plugin/src/components/ScheduleUserDetails/ScheduleUserDetails.tsx

```diff
@@ -1,39 +1,112 @@
 import { User } from 'models/user/user.types';
+import { useStore } from 'state/useStore';
 
 import styles from './ScheduleUserDetails.module.css';
 
 interface ScheduleUserDetailsProps {
   currentMoment: dayjs.Dayjs;
   user: User;
+  isOncall: boolean;
 }
 
 const cx = cn.bind(styles);
 
 const ScheduleUserDetails: FC<ScheduleUserDetailsProps> = (props) => {
-  const { user, currentMoment } = props;
+  const { user, currentMoment, isOncall } = props;
   const userMoment = currentMoment.tz(user.timezone);
   const userOffsetHoursStr = getTzOffsetString(userMoment);
+  const isInWH = isInWorkingHours(currentMoment, user.working_hours, user.timezone);
 
+  const store = useStore();
+
+  const { teamStore } = store;
+  let slackWorkspaceNameOrigin = teamStore.currentTeam.slack_team_identity.cached_name;
+  const slackWorkspaceName = slackWorkspaceNameOrigin.replace(/[^0-9a-z]/gi, '');
   return (
     <div className={cx('root')}>
-      <VerticalGroup spacing="sm">
+      <VerticalGroup spacing="md">
         <HorizontalGroup justify="space-between">
           <Avatar src={user.avatar} size="large" />
         </HorizontalGroup>
         <VerticalGroup spacing="sm">
           <Text type="primary">{user.username}</Text>
-          <Text type="secondary">
-            {`${userMoment.tz(user.timezone).format('DD MMM, HH:mm')}`} {userOffsetHoursStr}
-          </Text>
+          {isOncall && <Badge text="OnCall now" color="green" />}
+          {isInWH ? (
+            <Badge text="Inside working hours" color="blue" />
+          ) : (
+            <Badge text="Outside working hours" color="orange" />
+          )}
+          <HorizontalGroup align="flex-start">
+            <div className={cx('timezone-icon')}>
+              <Text type="secondary">
+                <Icon name="clock-nine" />
+              </Text>
+            </div>
+            <div className={cx('timezone-wrapper')}>
+              <div className={cx('timezone-info')}>
+                <VerticalGroup>
+                  <Text type="secondary">Local time</Text>
+                  <Text type="secondary">{currentMoment.tz().format('DD MMM, HH:mm')}</Text>
+                  <Text type="secondary">({getTzOffsetString(currentMoment)})</Text>
+                </VerticalGroup>
+              </div>
+
+              <div className={cx('timezone-info')}>
+                <VerticalGroup className={cx('timezone-info')}>
+                  <Text>{user.username}'s time</Text>
+                  <Text>{`${userMoment.tz(user.timezone).format('DD MMM, HH:mm')}`}</Text>
+                  <Text>({userOffsetHoursStr})</Text>
+                </VerticalGroup>
+              </div>
+            </div>
+          </HorizontalGroup>
+        </VerticalGroup>
+
+        <hr className={cx('hr')} />
+        <VerticalGroup spacing="sm">
+          <Text>Contacts</Text>
+
+          <div className={cx('contact-details')}>
+            <Text type="secondary">
```

Comment:
I've noticed we're styling the anchor links with `text-decoration-line: none;` - I haven't paid much attention to `Text`'s type but shouldn't it be `link` when we have anchor links within? :thinking: